d4rl.rst

```diff
@@ -0,0 +1,347 @@
+D4RL (Mujoco)
+~~~~~~~~~~~~~~
+
+Overview
+===========
+D4RL is an open-source benchmark for offline Reinforcement Learning that provides standardized environments and datasets for training and benchmarking algorithms. The collection strategy for the dataset includes
+
+1. Datasets generated by hand-designed rules and expert demonstrations
+2. Multi-task datasets (agents perform different tasks in the same environment)
+3. Datasets collected using mixed strategies
+
+Specifically, it includes the following 7 sub-environments
+
+- Maze2D
+- AntMaze
+- Gym-MuJoco
+- Adroit
+- FrankaKitchen
+- Flow
+- Offline CARLA
+
+Note: offline rl is a dataset of d4rl for training, and testing is to interact with a specific RL environment, such as Mujoco.
+
+The Mujoco dataset is a physics engine designed to promote research and development in fields such as robotics, biomechanics, graphics, and animation that require fast and accurate simulation. It is often used as a benchmarking environment for continuous space reinforcement learning algorithms. It is a collection of 20 sub-environments. In D4RL, the sub-environments used are Half Cheetah, Hopper, Walker2D.
+Each sub-environment contains 5 mini-environments
+
+- expert: train a \ `SAC <https://arxiv.org//abs/1801.01290>`__\ algorithm online until the strategy reaches the expert performance level, using the expert strategy to collect 1 million sample data
+- medium-expert: mixes equal amounts of data collected by expert and medium strategies
+- medium: first train a SAC algorithm online, stop training in the middle, and then use this partially trained strategy to collect 1 million sample data
+- medium-replay: trains a SAC algorithm online until the policy reaches a medium performance level, collecting all samples placed in the buffer during training
+- random: use a random initialization strategy to collect
+
+The picture below shows one of the Hopper games.
+
+.. image :: ./images/d4rl.gif
+   :align : center
+
+Install
+====
+
+Installation method
+--------
+Just install the d4rl, gym and mujoco-py libraries, in which d4rl can be installed with one click through pip or through clone
+
+.. code :: shell
+
+    # pip install
+    pip install git+https://github.com/rail-berkeley/d4rl@master#egg=d4rl
+
+    # installed by cloning the repository
+    git clone https://github.com/rail-berkeley/d4rl.git
+    cd d4rl
+    pip install -e .
+
+
+
+
+Mujoco only needs the gym and mujoco-py two libraries, which can be installed by one-click pip or combined with DI-engine
+
+1. The mujoco-py library no longer requires an activation license (``mujoco-py>=2.1.0``), you can install free-mujoco-py via \ `pip <https://github.com/openai/ mujoco-py/pull/640>`__ install

```

Comment:
1. The mujoco-py library no longer requires an activation license (``mujoco-py>=2.1.0``), you can \ `pip install free-mujoco-py <https://github.com/openai/mujoco-py/pull/640>`__  install